src/components/lyne-panel/lyne-panel.scss

```diff
@@ -0,0 +1,93 @@
+:host {
+  display: block;
+
+  --font-family: 'SBBWeb Light';
+  --letter-spacing: 0.01em;
+  --panel-width: 223;
+  --panel-height: 167;
+  --font-size: 24;
+  --line-height: 1.3;
+  --button-margin: var(--spacing-fixed-5x);
+  --panel-padding-inline: var(--spacing-responsive-s-medium);
+  --triangle-height: 33;
+
+  @include mq($from: large) {
+    --panel-width: 321;
+    --panel-height: 240.11;
+    --font-size: 32;
+    --line-height: 1.3;
+    --button-margin: var(--spacing-fixed-6x);
+    --panel-padding-inline: var(--spacing-responsive-m-medium);
+    --triangle-height: 47.45;
+  }
+
+  @include mq($from: ultra) {
+    --panel-width: 428;
+    --panel-height: 320.38;
+    --font-size: 40;
+    --line-height: 1.2;
+    --button-margin: var(--spacing-fixed-5x);
+    --panel-padding-inline: var(--spacing-responsive-l-medium);
+    --triangle-height: 63.31;
+  }
+
+  --background-color: var(--color-red-default);
+  --text-color: var(--color-white-default);
+
+  padding-block-start: pxToRem(var(--triangle-height));
+  padding-block-end: pxToRem(var(--triangle-height));
+
+}
+
+.lyne-panel {
+  box-sizing: border-box;
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: pxToRem(var(--panel-width));
+  min-height: pxToRem(var(--panel-height));
+  padding-inline-start: pxToRem(var(--panel-padding-inline));
+  padding-inline-end: pxToRem(var(--panel-padding-inline));
```

Comment:
Maybe we should add some padding-block-start and padding-block-end as well. If a lot of text is entered it comes very close to the triangle and does not look too nice.

Reply:
done